scripts/generate_json_api.py

```diff
@@ -66,43 +66,144 @@ class Examples(CommandTestCase):
     async def play(self):
         r = self.recorder
 
+        # general sdk
+
+        await r(
+            'Get Status',
+            'status'
+        )
         await r(
-            'List your accounts.',
+            'Get Version',
+            'version'
+        )
+        await r(
+            'Get Settings',
+            'settings', 'get'
+        )
+
+        # accounts
+
+        await r(
+            'List your accounts',
             'account', 'list'
         )
 
         account = await r(
-            'Create an account.',
+            'Create an account',
             'account', 'create', '"generated account"'
         )
 
         await r(
-            'Remove an account.',
-            'account', 'remove', account['id']
+            'Add an account from seed',
+            'account', 'add', '"new account"', f"--seed=\"{account['seed']}\""
```

Comment:
They are two accounts. The goal here was to demonstrate account add and account create. The two accounts have different names.  account['id'] is the account named "new account" while the account being added is named "generated account". I `account remove` "new account" near the end after I use it for send/fund. Am I missing something?

Reply:
Name of the account is not really used anywhere but the keys representing the account do matter (this is where the account_id comes from). In the example above, you create an account and then add the same account again, this creates two accounts with the same account_id. This should actually fail but it works to avoid a bricked wallet in case user manually edits their wallet file and adds the same account twice. Normally you should never add the same account twice.

Reply:
OH, you are very right. I will look at rearranging it.